### DISCUSSION TITLE
Automate WinGet Manifest Submission

### DISCUSSION THREAD

**Discussion by arpan3t** [PowerShell/PowerShell#23971] [Enhancements]
PowerShell MS docs recommended installation method is to use WinGet, but I've noticed that PowerShell releases are out for up to a couple days before the pull request to [WinGet-pkgs](https://github.com/microsoft/winget-pkgs/pull/158784) is made.  I get the new version release notification when launching a new Terminal or VS Code PowerShell extension loads, run `winget list --upgrade-available` or `winget search` and there's no update...   

I understand that WinGet has its own submission review process and I'm not looking for an up to the second orchestrated release, but using the 7.4.3 release as an example, an ~18-hour lapse between release and WinGet pull request could be better.  Not sure how this is currently handled, but I'd imagine a GitHub action or Azure pipeline could automate the manifest submission by triggering on release.

Obviously not a priority enhancement, just a thought that would help align with MS documentation.

**Comment by ThomasNieto** [comment]
Here is an issue for automating WinGet releases: #19927

**Reply by arpan3t** [reply]
Oh nice, thanks for creating the issue!  No movement in 11 months 😞
